synths/core-synths-global.scd

```diff
@@ -182,10 +182,16 @@ CORE SYNTHDEFS FOR DIRT
 		sound2 = HPF.ar(in, 800);
 		sound1 = in - sound2;
 		sound1 = leslie * (1.0 - throb) * sound1;
-		sound1 = Balance2.ar(sound1[0], sound1[1], bal1);
-		sound2 = DelayC.ar(sound2, 1, distance / 343);
-		sound2 = leslie * (1.0 - distance) * sound2;
-		sound2 = Balance2.ar(sound2[0], sound2[1], bal2);
+
+		if (numChannels > 1, {			
```

Comment:
here I see that it is necessary to distinguish the cases. Was it this that caused the troubles originally maybe?

Reply:
yes, and i am sorry for the formatting changes, i use sclang in emacs, so i am very tempted to use automatic formatting :-) 

if these cases are not present, superdirt won't launch when using previous startup file (given in the example above)